modules/customize-installation-operator-overlays.adoc

```diff
@@ -0,0 +1,192 @@
+:_mod-docs-content-type: CONCEPT
+[id="customize-installation-operator-overlays_{context}"]
+
+= Customizing the installation using the Operator with overlays
+
+[role="_abstract"]
+Learn how to tailor the installation of {product-title-short} using the Operator method with overlays.
+
+[id="overlays_{context}"]
+== Overlays
+
+When `Central` or `SecuredCluster` custom resources don't expose certain low-level configuration options as parameters,
+you can use the `.spec.overlays` field for adjustments. Use this field to amend the Kubernetes resources
+generated by these custom resources.
+
+The `.spec.overlays` field comprises a sequence of patches, applied in their listed order. These patches are processed
+by the Operator on the Kubernetes resources before deployment to the cluster.
+
+[WARNING]
+====
+The `.spec.overlays` field present in both `Central` and `SecuredCluster` allows users to modify
+low-level kubernetes resources in arbitrary ways. Use this feature only when the desired customization
```

Comment:
```suggestion
low-level Kubernetes resources in arbitrary ways. Use this feature only when the desired customization
```

Reply:
Kubernetes is capitalized per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#kubernetes)